src/Tribe/Integrations/WPML/Utils.php

```diff
@@ -59,7 +60,9 @@ public static function get_wpml_i18n_strings( array $strings, $locale = null, ar
 		// Get the strings on multiple Domains and Languages
 		// WPML filter is unhooked to avoid the locale being set to the default one
 		remove_filter( 'locale', array( $sitepress, 'locale_filter' ) );
-		$translations = tribe( 'tec.i18n' )->get_i18n_strings_for_domains( $strings, $languages, $domains, $current_locale );
+		$flags = I18n::COMPILE_INPUT;
```

Comment:
:warning: Equals sign not aligned with surrounding assignments; expected 8 spaces but found 1 space

<sub>hash: f59d096c4d18d199caf4bc8f8bb3a27e</sub>